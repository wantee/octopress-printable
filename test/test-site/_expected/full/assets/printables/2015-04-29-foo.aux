\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand*\HyPL@Entry[1]{}
\HyPL@Entry{0<</S/D>>}
\providecommand \oddpage@label [2]{}
\@writefile{toc}{\contentsline {section}{\numberline {1}Math}{1}{section.1}}
\newlabel{math}{{1}{1}{Math}{section.1}{}}
\newlabel{eq:ellipse}{{1}{1}{Math}{equation.1.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {2}Image}{1}{section.2}}
\newlabel{image}{{2}{1}{Image}{section.2}{}}
\@writefile{toc}{\contentsline {section}{\numberline {3}Tex2img}{1}{section.3}}
\newlabel{tex2img}{{3}{1}{Tex2img}{section.3}{}}
\@writefile{toc}{\contentsline {section}{\numberline {4}Gist}{1}{section.4}}
\newlabel{gist}{{4}{1}{Gist}{section.4}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces Figure Example}}{2}{figure.1}}
\newlabel{fig:xor}{{1}{2}{Figure Example}{figure.1}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {2}{\ignorespaces Test tex figure}}{2}{figure.2}}
\newlabel{fig:test}{{2}{2}{Test tex figure}{figure.2}{}}
\citation{mikolov2013efficient}
\bibstyle{unsrt}
\bibdata{assets/printables/references}
\bibcite{mikolov2013efficient}{{1}{}{{}}{{}}}
\providecommand\NAT@force@numbers{}\NAT@force@numbers
\@writefile{toc}{\contentsline {section}{\numberline {5}Post Link}{3}{section.5}}
\newlabel{post-link}{{5}{3}{Post Link}{section.5}{}}
