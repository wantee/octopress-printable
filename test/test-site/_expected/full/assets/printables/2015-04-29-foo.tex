\documentclass[]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
    \usepackage{xltxtra,xunicode}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Mapping=tex-text,Scale=MatchLowercase}
  \newcommand{\euro}{â‚¬}
\fi
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\ifxetex
  \usepackage[setpagesize=false, % page size defined by xetex
              unicode=false, % unicode breaks when used with xetex
              xetex]{hyperref}
\else
  \usepackage[unicode=true]{hyperref}
\fi
\hypersetup{breaklinks=true,
            bookmarks=true,
            pdfauthor={},
            pdftitle={Foo},
            colorlinks=true,
            citecolor=blue,
            urlcolor=blue,
            linkcolor=magenta,
            pdfborder={0 0 0}}
\urlstyle{same}  % don't use monospace font for urls
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\setcounter{secnumdepth}{5}

\title{Foo}
\date{}
\usepackage{graphicx}
\usepackage[all]{hypcap}
\usepackage[sort&compress, numbers]{natbib}
\usepackage{minted}

\begin{document}
\maketitle

\section{Math}\label{math}

\begin{equation}\label{eq:ellipse}
Ax^2 + Bxy + Cy^2 + Dx + Ey + F = 0
\end{equation}

Refering equation \eqref{eq:ellipse}, \(\forall n, x_i^n = x_i\).

\section{Image}\label{image}

See following figure(\autoref{fig:xor}).

\begin{figure}[h]\centering\includegraphics[width=\textwidth]{.//assets/images/xor.png}\caption{Figure Example}\label{fig:xor}\end{figure}

\section{Gist}\label{gist}

\inputminted[mathescape, linenos, frame=lines, framesep=2mm]{Python}{.gist-cache/cache.7aeefc0de1bb10005514355a5c4a5dfe-1c74b8336494cb0e9c6d-xor-5d.py}

\section{Post Link}\label{post-link}

\href{http://localhost:4000/2015/04/29/foo/}{Foo}

Awesome paper\cite{mikolov2013efficient}.

\bibliographystyle{unsrt}

\bibliography{assets/printables/references}

\renewcommand{\thefootnote}{}

\footnotetext{\url{http://localhost:4000//2015/04/29/foo/}}

\end{document}
