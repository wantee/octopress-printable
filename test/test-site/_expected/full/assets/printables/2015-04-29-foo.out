\BOOKMARK [1][-]{section.1}{Math}{}% 1
\BOOKMARK [1][-]{section.2}{Image}{}% 2
\BOOKMARK [1][-]{section.3}{Gist}{}% 3
\BOOKMARK [1][-]{section.4}{Post\040Link}{}% 4
